core/integration/module_call_test.go

```diff
@@ -1920,7 +1920,7 @@ func (CallSuite) TestByName(ctx context.Context, t *testctx.T) {
 			`,
 			).
 			WithWorkdir("/work").
-			With(daggerExec("init")).
+			With(daggerExec("init", "--source=.")).
```

Comment:
suggestion: could we switch to using `With(withModInit)` everywhere we do `init`?

I've wanted to do this for a while, and it'll make changes where we have to update `init`s everywhere a lot easier. That said, maybe we should merge this one quickly, and maybe follow-up with some test cleanups? I can make a separate issue (cc @marcosnils).

Reply:
Yeah, IMO we should do a follow up for this change. But also not strongly opposed to that if you insist :)